app/test_main.py

```diff
@@ -1 +1,64 @@
-# write your code here
+from unittest import mock
+
+import pytest
+
+from app.main import can_access_google_page
+
+
+@pytest.fixture()
+def mocked_valid_google_url() -> None:
```

Comment:
`None` is incorrect type hint, because this function returns `MagicMock` objects